### PR TITLE
[Frontend] Payment Details

### PR DESCRIPTION
**Pull request by KezTimple**
## Ticket/s

[-Checkout Page (Payment Details)](https://trello.com/c/WEaB8O5C/98-checkout-page-payment-details)

## Changes

- Add payment details component

## Screenshot

image

## How to test?

- run pnpm dev
- Insert to url http://localhost:5173/
- Click Open Modal Button

### CODE REVIEW

File: packages/ui/Card/OrderSummary.tsx [hunk 1]
```diff
@@ -77,6 +79,7 @@ export const OrderSummary = () => {
           </Group>
         </Card.Section>
       </Card>
+      <PaymentDetails totalPrice={totalPrice} deliveryFee={deliveryFee} />
```
Comment (TreshAnn) [inline packages/ui/Card/OrderSummary.tsx:82]:
```suggestion
      <PaymentDetails totalPrice={totalPrice} deliveryFee={deliveryFee} />
      <Button fw={400} fullWidth style={{ color: 'black' }}>
        Place Order
      </Button>
```

File: packages/ui/Card/PaymentDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+              <IoCashOutline size={20} style={{ marginRight: '5px' }} />
+              <Text size="sm">Cash Payment</Text>
+            </Flex>
+          </div>
+        </Group>
+      </Card.Section>
+
+      <Card.Section p="xs" withBorder>
+        <Group position="apart">
+          <Text fw={700}>TOTAL</Text>
```
Comment (TreshAnn) [inline packages/ui/Card/PaymentDetails.tsx]:
```suggestion
          <Text fw={400}>Total:</Text>
```

**Review by TreshAnn — CHANGES_REQUESTED** [review]

File: packages/ui/Card/OrderSummary.tsx [hunk 3]
```diff
@@ -1,5 +1,7 @@
 import { Card, Flex, Grid, Group, Paper, Text } from '@mantine/core';
```
Comment (TreshAnn) [inline packages/ui/Card/OrderSummary.tsx]:
```suggestion
import { Card, Flex, Grid, Group, Paper, Text, Button} from '@mantine/core';

```

import button

**Review by TreshAnn — CHANGES_REQUESTED** [review]

**Review by TreshAnn — APPROVED** [review]

**Review by sandrex-SGV — APPROVED** [review]
LGTM

**Pull request merged**